src/setup/serverless-config.go

```diff
@@ -149,19 +204,17 @@ func DeployService(path string) string {
 	return slsDeployMessage
 }
 
-// GetEndpointID scrapes the serverless deploy message for the endpoint ID
-func GetEndpointID(slsDeployMessage string) string {
-	lines := strings.Split(slsDeployMessage, "\n")
-	if lines[1] == "endpoints:" {
-		line := lines[2]
-		link := strings.Split(line, " ")[4]
-		httpId := strings.Split(link, ".")[0]
-		endpointId := strings.Split(httpId, "//")[1]
-		return endpointId
-	}
-	line := lines[1]
-	link := strings.Split(line, " ")[3]
-	httpId := strings.Split(link, ".")[0]
-	endpointId := strings.Split(httpId, "//")[1]
+// GetEndpointIDAWS finds the AWS endpoint ID from the deployment message
+func GetEndpointIDAWS(slsDeployMessage string) string {
+	regex := regexp.MustCompile(`https:\/\/(.*)\.execute`)
+	return regex.FindStringSubmatch(slsDeployMessage)[1]
+}
+
+// GetEndpointIDAzure finds the Azure endpoint ID from the deployment message
+func GetEndpointIDAzure(message string) string {
```

Comment:
The GCR equivalent of this function is named `GetGCREndpointID`, you can amend it to `GetEndpointIDGCR` during rebase for consistency.